docker/aif-validator/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM amazonlinux:latest
+MAINTAINER craig.warsaw@caci.com
```

Comment:
Should Craig's email address be changed to yours? I also saw Craig referenced in the AIDA-Interchange-Format/README.md.

Reply:
Good catch. I've removed MAINTAINER as it's deprecated and I've changed README.md to use aif-support@nextcentury.com